packages/protobuf/src/registry.ts

```diff
@@ -41,6 +42,9 @@ export const typeRegistry: IMessageTypeRegistry = createRegistry(
   // gen/ibc/core/channel/v1/tx_pb
   MsgRecvPacket,
 
+  // gen/ibc/core/client/v1/tx_pb
+  MsgUpdateClient,
+
```

Comment:
Required to add the the type registry as an IbcRelay transaction sent from the extension to the frontend needs to be able to serialize the `Any` inside of it & a `MsgUpdateClient` is accompanied with a `MsgRecvPacket`